site/layouts/shortcodes/minikube-k8s-minor-version.html

```diff
@@ -1,5 +1,5 @@
-{{- if eq (getenv "HUGO_ENV") "production" }}
+{{- if or (eq (getenv "HUGO_ENV") "production") (eq (getenv "HUGO_ENV") "snapshot")}}
 {{- $.Page.Site.Params.minikube_minor_supported_k8s }}
 {{- else }}
 {{- $.Page.Site.Params.dev_minikube_minor_supported_k8s }}
-{{- end }}
```

Comment:
I don't think the line break is causing an issue right now, but I'd still be tempted to remove the linebreak anyway just to prevent it from happening anyway.